_posts/2023-01-02-release.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Release 2023.01
+author: priya
+date: 2023-01-02
+category: Jekyll
+layout: post
+---
+
+### Defect fixes
+
+* Column browser was enabled when there was a pipeline error.
+* Pipeline in error -Tooltip message was not showing up on hover for disabled items.
+* Appropriate error message was not appearing when user tries to delete foreign view.
```

Comment:
Added an error message when the user tries to delete a View which has been used in a join rule elsewhere